octoprint_mrbeam/migrate.py

```diff
@@ -264,12 +276,74 @@ def run(self):
             else:
                 self._logger.debug("No migration required.")
 
+            self._run_migration()
             self.save_current_version()
         except MigrationException as e:
             self._logger.exception("Error while migration: {}".format(e))
         except Exception as e:
             self._logger.exception("Unhandled exception during migration: {}".format(e))
 
+    def _run_migration(self):
```

Comment:
What qould be the benefit? For me it makes more sense here as the migrate.py is reponsible for executing the migrations

Reply:
The gained benefir is that we have the logic for the new migration process all under one folder, and when you try to run it, you will always use the base class so it would be more clear in the code